project/Build.scala

```diff
@@ -1026,6 +1026,8 @@ object Build {
        * flag prevents these mistakes from happening.
        */
       scalacOptions += "-Yno-predef",
+      // We also opt in to emitting static forwarders for non-top-level objects
```

Comment:
IMO this comment is not helpful. It describes *what* the code does not *why* it does it. I think a comment explaining why is warranted here.